pkg/registry/apis/folders/register.go

```diff
@@ -165,14 +168,20 @@ func (b *FolderAPIBuilder) GetAuthorizer() authorizer.Authorizer {
 			}
 
 			action := dashboards.ActionFoldersRead
-			scope := dashboards.ScopeFoldersProvider.GetResourceScopeUID(attr.GetName())
+			scope := dashboards.ScopeFoldersProvider.GetResourceScopeUID(name)
 
 			// "get" is used for sub-resources with GET http (parents, access, count)
-			switch attr.GetVerb() {
+			switch verb {
```

Comment:
You to swap position of patch case with create, patch should use dashboards.ActionFoldersWrite